README.adoc

```diff
@@ -1,85 +1,90 @@
 = IoT OpenShift Example Project
 ===============
 
-This example showcases Intelligent IoT Gateway on the Red Hat OpenShift Container Platform. The Intelligent IoT Gateway enables real time decision-making at the edge. OpenShift provides secure enterprise-grade container platform to accelerate IoT application delivery. Through the use of containers and microservices, this example enables security, scalability and ease of deployment for IoT use case.
+This example showcases an Intelligent Internet-of-Things (IoT) Gateway on https://www.openshift.com/container-platform/[Red Hat's OpenShift Container Platform].  The Gateway enables real-time decision making at the edge, and demonstrates the ease with which you can deploy a secure and scalable microservice application on the enterprise-grade OpenShift application platform.
 
-
-image::images/IoT-OSE-Containers.png[width="462", height="614", align=center]
+image::images/IoT-OSE-Containers.png[height="400", align="center"]
 
 *Please note: This repository is currently under development and the feature set/components are subject to change without notice*
 
 == Components
 
-- *https://access.redhat.com/documentation/en/red-hat-xpaas/version-0/red-hat-xpaas-fuse-integration-services-image/[JBOSS Fuse Integration Services]*
- A set of tools and container images that enable development, deployment, and management of microservices within OpenShift.
-- *https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/[JBOSS A-MQ]*
-A-MQ message broker container image for communicating with Sensor using MQTT protocol
-- *https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-decision-server-image/chapter-1-introduction-to-the-decision-server-image/[JBOSS BRMS Decision Server]*
-Container image of the decision platform for executing realtime business rules
-- *https://zeppelin.apache.org/[Apache Zeppelin]*
-Container image of the tool for interactive data analytics.
-- *https://www.postgresql.org/[PostgreSQL]*
-Container image of the database for storing sensor data
-- *Simulated Software Sensor*
-Spring boot based software app simulating temperature, vibration and GPS sensor data
+The application is deployed as a set of microservices inside containers on OpenShift:
+
+- *https://access.redhat.com/documentation/en/red-hat-xpaas/version-0/red-hat-xpaas-fuse-integration-services-image/[JBoss Fuse Integration Services]* (FIS): Tools that enable rapid development, deployment, and management of microservices within OpenShift.
```

Comment:
FIS provides a set of tools to facilitate integration, routing, transformation etc (with an emphasis using Apache Camel). I would highlight this aspect of the FIS featureset